simulation/tools/sim_viewer.cpp

```diff
@@ -96,7 +96,7 @@ using GeometryHandlerConstPtr = GeometryHandler::ConstPtr;
 #define NORMALS_SCALE 0.01
 #define PC_SCALE 0.001
 
-std::uint16_t t_gamma[2048];
+std::uint16_t t_gamma[2048]{};
```

Comment:
Why did you add this?

Reply:
Zero-initialization. In this case (per Godbolt) it's only a single instruction:

`t_gamma:
  .zero 4096`

Reply:
But `t_gamma` is filled with the desired values here: https://github.com/PointCloudLibrary/pcl/blob/master/simulation/tools/sim_viewer.cpp#L844 So zero-initialization doesn't seem necessary?

Reply:
Fair enough, reverted. The decl is a long way from the point of first use, though...